server/chat-formatter.ts

```diff
@@ -167,7 +156,8 @@ class TextFormatter {
 
 		this.pushSlice(start);
 		while (this.stack.length > stackPosition + 1) this.popSpan(start);
-		const span = /** @type {FormatSpan} */ (this.stack.pop());
+		const span = this.stack.pop();
```

Comment:
actually, we know `this.stack.length` > 1 because `stackPosition` >= 0,so I think you can just do `const span = this.stack.pop()!` instead and remove the early return

Reply:
Hm, yeah that works.